Test/Web/FunctionalTests/TestApps/WebAppFW45/packages.config

```diff
@@ -1,11 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.ApplicationInsights" version="2.4.0-beta1-build07987" targetFramework="net451" />
   <package id="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" version="2.4.0-beta1-build07987" targetFramework="net451" />
+  <package id="Microsoft.AspNet.Diagnostics" version="1.0.0-beta-10405-01" targetFramework="net451" />
```

Comment:
>"1.0.0-beta-10405-01 [](start = 53, length = 20)

why the version of this module is not aligned with the rest of ASP.NET? Should it be 4.7 something?